int16 obj_id
float64[6] state_vecs
